### PR TITLE
fix: fix code warning, and do the code format

### PR DESCRIPTION
**Pull request by deer-hang**
**What this PR does / why we need it**:

- fix code warning, and do the code format
- use go-multierror package in obs service

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/obs' TESTARGS='-run TestAccObsBucketObjectDataSource_content'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/obs -v -run TestAccObsBucketObjectDataSource_content -timeout 360m -parallel 4 
=== RUN   TestAccObsBucketObjectDataSource_content 
=== PAUSE TestAccObsBucketObjectDataSource_content
=== CONT  TestAccObsBucketObjectDataSource_content
--- PASS: TestAccObsBucketObjectDataSource_content (14.89s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/obs       14.943s



make testacc TEST='./huaweicloud/services/acceptance/obs' TESTARGS='-run TestAccDataSourceObsBuckets_basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/obs -v -run TestAccDataSourceObsBuckets_basic -timeout 360m -parallel 4 
=== RUN   TestAccDataSourceObsBuckets_basic 
=== PAUSE TestAccDataSourceObsBuckets_basic
=== CONT  TestAccDataSourceObsBuckets_basic
--- PASS: TestAccDataSourceObsBuckets_basic (11.21s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/obs       11.259s


make testacc TEST='./huaweicloud/services/acceptance/obs' TESTARGS='-run TestAccObsBucketObject_source'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/obs -v -run TestAccObsBucketObject_source -timeout 360m -parallel 4 
=== RUN   TestAccObsBucketObject_source 
=== PAUSE TestAccObsBucketObject_source
=== CONT  TestAccObsBucketObject_source
--- PASS: TestAccObsBucketObject_source (15.84s) 
PASS 
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/obs       15.893s


 make testacc TEST='./huaweicloud/services/acceptance/obs' TESTARGS='-run TestAccObsBucketPolicy_basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/obs -v -run TestAccObsBucketPolicy_basic -timeout 360m -parallel 4 
=== RUN   TestAccObsBucketPolicy_basic 
=== PAUSE TestAccObsBucketPolicy_basic 
=== CONT  TestAccObsBucketPolicy_basic
--- PASS: TestAccObsBucketPolicy_basic (10.50s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/obs       10.548s 



make testacc TEST='./huaweicloud/services/acceptance/obs' TESTARGS='-run TestAccObsBucket_basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/obs -v -run TestAccObsBucket_basic -timeout 360m -parallel 4 
=== RUN   TestAccObsBucket_basic 
=== PAUSE TestAccObsBucket_basic
=== CONT  TestAccObsBucket_basic
--- PASS: TestAccObsBucket_basic (26.14s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/obs       26.210s 

```

### CODE REVIEW

File: huaweicloud/services/obs/resource_huaweicloud_obs_bucket.go [hunk 1]
```diff
@@ -1456,7 +1456,7 @@ func getObsError(action string, bucket string, err error) error {
 
 // normalize format of storage class
 func normalizeStorageClass(class string) string {
-	var ret string = class
+	var ret = class
```
Comment (ShiChangkuo) [inline huaweicloud/services/obs/resource_huaweicloud_obs_bucket.go]: `var ret string = class` is using more in golang.

Comment (deer-hang) [reply huaweicloud/services/obs/resource_huaweicloud_obs_bucket.go]: OK, it has been fixed

File: huaweicloud/services/obs/resource_huaweicloud_obs_bucket.go [hunk 2]
```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/hashicorp/go-multierror"
```
Comment (ShiChangkuo) [inline huaweicloud/services/obs/resource_huaweicloud_obs_bucket.go]: it's better to import the package in L12

**Pull request merged**